### DISCUSSION TITLE
Bad Weight Suggestion

### DISCUSSION THREAD

**Discussion by Cotrox** [zepp-health/zeppos-forum#384] [User Feedback]
Zepp suggest me 70kg but i’m 1,80m

**Comment by MyoungXUE** [comment]
Is the weight suggestion provided by Zepp? I find it reasonable. The scientific basis should be BMI.

**Reply by Cotrox** [reply]
https://m.my-personaltrainer.it/peso-forma.html

Put 1.80
Age 25

BMI is 70 but suggested weight is 78-80

**Reply by MyoungXUE** [reply]
Zepp's reference is BMI, for reference only. Everyone knows their own body best, and I believe you can develop the most suitable plan for yourself.

**Comment by rubenferb** [comment]
body weight suggestion are completely off in your app for any person that has a little muscle. Being a professional athlete and seeing you are overweight or your ideal weight is 20 kilos below your actual weight its offensive and dangerous. I didnt even weight that when i was a skinny teenager. You are suggesting a range of weight that starts at 58 for someone who is 180cm. BMI has been debunked by now and should not be used to make health assestments on weight.
